lance/lance-rest-server/src/main/java/org/apache/gravitino/lance/service/rest/LanceTableOperations.java

```diff
@@ -176,11 +195,34 @@ public Response deregisterTable(
       @Context HttpHeaders headers,
       DeregisterTableRequest deregisterTableRequest) {
     try {
+      validateDeregisterTableRequest(deregisterTableRequest);
       DeregisterTableResponse response =
           lanceNamespace.asTableOps().deregisterTable(tableId, delimiter);
       return Response.ok(response).build();
     } catch (Exception e) {
       return LanceExceptionMapper.toRESTResponse(tableId, e);
     }
   }
+
+  private void validateCreateEmptyTableRequest(CreateEmptyTableRequest request) {
+    Preconditions.checkArgument(
+        StringUtils.isNotBlank(request.getLocation()), "Table location must be provided.");
```

Comment:
per OAS, the `location` is optional

Reply:
> per OAS, the location is optional

What's does it means?  the location can be null for an empty table?

Reply:
yes, If I am wrong, please correct me: https://github.com/lancedb/lance-namespace/blob/b035daec6b7cce0bec484c18326100d07c9e33bb/docs/src/rest.yaml#L2513-L2517

Reply:
Did you handle it and remind users that they can set the location in the schema and catalog? Currently, Gravitino does support the feature you address, but I'm worried that it's not the design that Lance REST supports.

Reply:
you should follow the Lance REST OAS